pkg/server/commands/acquire.go

```diff
@@ -68,6 +69,8 @@ func (c *acquireCmd) Run() error {
 		}
 	}
 
+	rand.Shuffle(len(cirs), func(i, j int) { cirs[i], cirs[j] = cirs[j], cirs[i] })
```

Comment:
Seems a good point, I think the same effect could be achieved by using (below) just `sort.Slice` instead of `sort.SliceStable` (see https://pkg.go.dev/sort#Slice) since it's not guaranteed to be stable